src/framework/actions/helpers.ts

```diff
@@ -27,3 +29,10 @@ export const makeInstance = (
   $.instances[I.id] = I;
   return I;
 };
+
+export const giveGold = (
```

Comment:
I suppose my worker code should use this helper function. I might just copy it over to there, but that would cause merge conflicts later.